Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
+            while buf = rd.gets
+              log.puts buf
+              print "."
```

Comment:
Currently, it will print a dot for every line of logs. If you want, we can make it every N lines or every N bytes.

>  it should probably have a timer and print e.g. no more than once a minute.

I kinda like dot printing corresponding real log. So we can see wether the build is really freezing.